src/shared/ProgressButtonsInner.js

```diff
@@ -87,7 +87,7 @@ const ProgressButtonsInner = ({
             onClick={() => (progressRedux === 4 ? setMyCoverCropActivationFlag() : changeProgress('increment'))}
             disabled={isDisabledNext || (progressRedux === 4 && selectedCropIdsRedux.length === 0)}
           >
-            {progressRedux === 4 ? 'VIEW MY LIST' : 'NEXT'}
+            {progressRedux === 4 ? 'MY SELECTED CROPS' : 'NEXT'}
```

Comment:
changing this text has left the button with 2 rows of text.  Can you please adjust the size of the button so we can see how it looks when the text is on just one line.  I can see we are adjusting the size of the button for this text already i just think it needs to be a little bigger so the text is on one line.